requirements.txt

```diff
@@ -4,3 +4,6 @@ setuptools>=19.2
 libclang-py3==0.2
 validators~=0.10
 appdirs~=1.4.0
+# Stable version 3.2 is problematic in Windows.
+# Use >=3.3 once nltk-3.3 is released.
+nltk~=3.1.0
```

Comment:
nope that one doesn't belong here :D
isn't it installed automatically anyway with `coala-bears`?